lib/std/fs/file.zig

```diff
@@ -60,6 +64,10 @@ pub const File = struct {
         /// `error.FileAlreadyExists` to be returned.
         exclusive: bool = false,
 
+        /// Ensure that this open call creates the file, otherwise causes
```

Comment:
Looks like these doc comments need to be updated

Reply:
Oh, actually, I need to remove that. I started working on adding locking, then decided to work on exposing the inode number instead. I'm planning on making a separate pull request for file locking